Automattic/wp-calypso
# Domains: Fix ownership filtering in the domains list

#### Changes proposed in this Pull Request
Depends on D70167-code. 
Fixes the ownership filtering in the domains list. It uses the `currentUserIsOwner` prop added in the back-end related changes.

#### Testing instructions
Go to a site's domains list and check that the ownership filtering (owner by me, owned by others) works as expected.
